### PR TITLE
fix: modify style and i18n

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
1. modify i18n
2. remove useless Drawer in trace search
3. modify the height of span chart in waterfall

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/locales/zh.json [hunk 1]
```diff
@@ -2300,9 +2300,11 @@
     "total span time": "总 span 时间",
     "trace id": "追踪ID",
     "trace times": "追踪次数",
+    "tracing": "链路追踪",
     "tracing debug": "链路调试",
     "tracing details": "链路详情",
     "tracing information": "链路信息",
+    "tracing query": "链路追踪",
```
Comment (McDaddy) [inline shell/app/locales/zh.json:2307]: `"tracing": "链路追踪",`
`"tracing query": "链路追踪",`  what's the difference?

Comment (sherotree) [reply shell/app/locales/zh.json:2307]: Backend requirements

Comment (sherotree) [reply shell/app/locales/zh.json:2307]: image
image

Comment (McDaddy) [reply shell/app/locales/zh.json:2307]: why two words? can it be one En word?

Comment (sherotree) [reply shell/app/locales/zh.json]: image

Change Chinese translation

**Pull request merged**